README.rst

```diff
@@ -15,21 +15,14 @@ Parsel
    :alt: Coverage report
 
 
-Parsel is a library to extract data from HTML and XML using XPath and CSS selectors
+Parsel is a BSD-licensed Python_ library to extract data from HTML_ and XML_
+using XPath_ and CSS_ selectors, optionally combined with
+`regular expressions`_.
 
-* Free software: BSD license
-* Documentation: https://parsel.readthedocs.org.
```

Comment:
I'm curious as to why you want to remove this line.

The main reason it is there is for people reading the README on the Github project page to quickly find the docs.

Reply:
It’s not removed, it’s moved below the quick example code.

After the change, the documentation looks like:
    <description>
    <quick example code>
    <link to the online documentation>

Reply:
To see the final result better, I suggest you check out [the GitHub rendering of the README.rst file on my branch](https://github.com/Gallaecio/parsel/tree/documentation-introduction#parsel).

Reply:
I know it is a matter of preference, but for me having a link to docs at the top also sounds better.